LeagueToolkit/IO/NVR/NVRVertexBuffer.cs

```diff
@@ -14,6 +14,7 @@ public class NVRVertexBuffer
 
         //Used for write stuff
         public NVRVertexType Type { get; private set; }
+        //Replaced by NVRDrawIndexedPrimitive.Vertices
```

Comment:
What is this comment supposed to mean exactly? Where is this replacement happening?